src/DataDryad.jl

```diff
@@ -0,0 +1,42 @@
+immutable DataDryad <: DataRepo
+end
+
+base_url(::DataDryad) = "https://datadryad.org/mn/"
+
+function description(::DataDryad, mainpage)
+    desc = replace(text_only(first(matchall(sel".article-abstract", mainpage.root))), "Abstract ", "")
+    author = text_only(first(matchall(sel".pub-authors a", mainpage.root)))
```

Comment:
an Extracting out just the first author and then saying `$(author) et. al.` 
is a good first approximation of the correct thing to say,
but it is not quiet correct.

This isn't blocking as far as merging this goes.
But it we merge it like it is will need to open an issue to remind us to fix it later.
(which is fine)

The correct is to do something like as described in
http://blog.apastyle.org/apastyle/2011/11/the-proper-use-of-et-al-in-apa-style.html

Reply:
A separate logic will have to be written for this. Perhaps as you said, I'll file a separate issue for this.